### PR TITLE
Created Result page

### PR DESCRIPTION
**Pull request by rutujadhanawade**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/HakinCodes/Malaria-Detection/blob/master/CONTRIBUTING.md)?

YES

### Description

 Created the Results Page using HTML, CSS, Bootstrap, JQuery. By using this [UI Design](https://github.com/HakinCodes/Malaria-Detection/blob/master/design/webpage_designs/result_page.png) .

### Checklist

- [x] I've read the contribution guidelines.
- [x] This PR mets all the requirements like Clean Code, proper README and Documentation

### Related Issues or Pull Requests

Fixes: #125 

### Screenshots (if any)

op1

### CODE REVIEW

**Comment by kahanikaar** [comment]
The Frontend looks good, do just check once the alignment responsiveness of the buttons.

**Review by kahanikaar — COMMENTED** [review]
The Frontend looks good, do just check once the alignment responsiveness of the buttons.

**Comment by rutujadhanawade** [comment]
Hi @kahanikaar buttons are responsive. But the images are not actually responsive. I used the max width property but still it is not showing much difference. Will you give me suggestion to make them responsive.

**Comment by kahanikaar** [comment]
> @kahanikaar buttons are responsive.

A suggestion, try using bootstrap containers, will be better in making it responsive, and also try using relative values for css fields.

**Comment by rutujadhanawade** [comment]
@kahanikaar Please review this. I have made all the necessary changes.

File: templates/result.html [hunk 1]
```diff
@@ -3,13 +3,42 @@
...
+ 
+ <img id="logo" src="{{url_for('static', filename='images/logo.png')}}">
+<div class="container">
+ <div class="row">
+ <img id="heading" src="{{url_for('static', filename='images/result4.png')}}">
+  </div>
+<!--    <h1><span> Result </span></h1>-->
+ <div class="row">
+ <div class="col-sm">
+    <button  type="button" class="imageBtn"><b>Maleria Cell</b></button>
```
Comment (kahanikaar) [inline templates/result.html:26]: Check the typo of malaria here

**Review by kahanikaar — CHANGES_REQUESTED** [review]
The rest frontend looks good. Aligns with the requirement.

**Comment by rutujadhanawade** [comment]
Fixed typo @kahanikaar

**Review by kahanikaar — APPROVED** [review]
Looks good to me. Approved from my side. Do review @HarshCasper @sarthakkundra @BALaka-18

**Review by sarthakkundra — APPROVED** [review]
Looks good! 👍

**Review by BALaka-18 — APPROVED** [review]
Looks good to me.

**Comment by sarthakkundra** [comment]
@BALaka-18 I have merged the restyled PR you can add the respective points.

**Pull request closed without merging**